gridpath/transmission/operations/carbon_emissions.py

```diff
@@ -307,6 +340,36 @@ def load_model_data(
         None: list(data_portal.data()["tx_carbon_cap_zone"].keys())
     }
 
+    # Check if timepoint emissions file exists before loading
+    timepoint_emissions_file = os.path.join(
+        scenario_directory,
+        weather_iteration,
+        hydro_iteration,
+        availability_iteration,
+        subproblem,
+        stage,
+        "inputs",
+        "transmission_timepoint_emissions.tab",
+    )
+
+    # If the timepoint emissions file exists, load the data and initialize the CRB_TX_OPR_TMPS set
+    if os.path.exists(timepoint_emissions_file):
+        data_portal.load(
+            filename=timepoint_emissions_file,
+            select=(
+                "transmission_line",
+                "timepoint",
+                "tx_co2_intensity_tons_per_mwh_hourly",
+            ),
+            param=(m.tx_co2_intensity_tons_per_mwh_hourly,),
+        )
+
```

Comment:
The CRB_TX_OPR_TMPS is already initialized in the model and should not be overwritten here. The concept of operational timepoints is a critical one in GridPath and is managed internally based on when projects can exist.